node-module-system/calculate.ts

```diff
@@ -0,0 +1,28 @@
+import { sum } from './add.js';
+import { difference } from './subtract.js';
+import { multiply } from './multiply.js';
+import { divide } from './divide.js';
+
+const operation: string = process.argv[3];
+
+if (operation === 'plus') {
+  console.log(
+    'result:',
+    sum(parseInt(process.argv[2]), parseInt(process.argv[4]))
+  );
+} else if (operation === 'minus') {
+  console.log(
+    'result:',
+    difference(parseInt(process.argv[2]), parseInt(process.argv[4]))
+  );
+} else if (operation === 'times') {
+  console.log(
+    'result:',
+    multiply(parseInt(process.argv[2]), parseInt(process.argv[4]))
+  );
+} else if (operation === 'divide') {
+  console.log(
+    'result:',
+    divide(parseInt(process.argv[2]), parseInt(process.argv[4]))
+  );
+}
```

Comment:
Good solution. Some suggestions:
- You repeat `process.argv[2]`, etc., several times. It would be nice to avoid that code duplication.
- You repeat `parseInt` several times. It would be nice to avoid that code duplication.
- You should use `parseFloat` instead of `parseInt` since `parseInt` won't handle numbers like `2.3` correctly.
- It's probably best to use `Number(x)` instead of `parseInt` or `parseFloat`, but that's just a convention.
- You repeat the formatting of the result several times. Code is easier to modify if you don't repeat yourself (this is called the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle).